### ISSUE TITLE
[Nvidia Shield Pro] Sync doesn't work altough the plugin is installed correctly

### ISSUE THREAD

**Issue by Smoukus** [trakt/Plex-Trakt-Scrobbler#552]
I am running Plex server on my Shield Pro. I installed the Plugin on the Shield and in the Web interface, the plugin is being recognized and I have configured it. I installed it properly because I have also installed other plugins that work flawlessly.
Whatever movie or series I watch, the progress isn't being  synced with Trakt.tv. I can't even see on Trakt.tv's website that I'm currently watching XY movie/series.

That worked without issues when I was running Plex server on my PC and when I had the plugin installed there.

Any help?

**Comment by agneevX** [comment]
You should install Kitana to manage the plugin and check the config.

**Comment by TonioBDS** [comment]
Hello everybody,
I think I have a similar issue on Nvidia Shield Pro, but unfortunately I couldn't find any clue of how to resolve it on the internet.
Is anyone so kind to let me know if it is finaly possible to make Trakt working and provide a step by step how-to ?

My conf is the following:

Shield Pro (2019) with PMS on shield, media files on my Synology NAS (DS218+) and metadata on high speed USB3-microSD (128Go)
Trakt.tv and webtools plugin files are copied on "Android\data\com.plexapp.mediaserver.smb\Plex Media Server\Plug-ins" folder
Access of Trakt by the means of Kitana with the following errors:
Trakt.tv: Critical Error: Unable to import the playhouse.apsw_ext library (dlopen failed: couldn't map "/storage/0123-4567/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so" segment 0: Permission denied)
Please let me know what you think about it.
Many thanks

**Comment by agneevX** [comment]
> segment 0: Permission denied)
`chown` to `plex` and see what happens.

**Comment by TonioBDS** [comment]
> > segment 0: Permission denied)
> 
> `chown` to `plex` and see what happens.

Thank you for your support agneevX.
Being not very familiar with Linux (or Unix) commands, I have question about how to do that.

I understand the idea is to modify the right permission of the plugin folder(s) to the correct "user".
However I have no idea how to do it.

I managed to install on my Nvidia shield Pro the application SSHelper.
SSHelper is supposed to allow me to connect my shield throuh SSH. I managed to connect but never to use chmod or chown command (maybe because I don't really understand what I do when I follow forum tips).
SSHelper also provides a terminal that I can theoretically use to type my commands directly on my shield (if I manage to connect a keyboard on it).

This is where I am.
Many thanks

**Comment by agneevX** [comment]
> Being not very familiar with Linux (or Unix) commands, I have question about how to do that.

Sorry I had no idea. 

What does `ls -l "/storage/0123-4567/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so"` output?

**Comment by TonioBDS** [comment]
Do you mean that I try to connect through SSH and then I enter this command ?
I suppose I have to adapt the path name ?

**Comment by agneevX** [comment]
Yes. Connect via SSH, paste that command and hit enter and paste the contents here.

**Comment by TonioBDS** [comment]
Please findthe result:
`u0_a103@localhost:~$ ls -l "/storage/0123-4567/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so"
-rwxr-x---    1 u0_a97   everybod   3008688 Feb 16 19:16 /storage/0123-4567/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so
`

However I am not sure it is the right path since my metadata are located on a USB stick at this location:\\SHIELD\Clé USB SanDisk\Android\data\com.plexapp.mediaserver.smb\Plex Media Server\Plug-ins\Trakttv.bundle\Contents\Libraries\Android\aarch64\ucs2

**Comment by TonioBDS** [comment]
Second try : I went through all the path manually to find the Plugin folder: 
Using my file manager I can access to the path "\SHIELD\Clé USB SanDisk\Android\data\com.plexapp.mediaserver.smb\Plex Media Server\Plug-ins\Trakttv.bundle\Contents\Libraries\Android\aarch64\ucs2" without trouble.

However in SSH when I get to "\SDCard\Android\data\com.plexapp.mediaserver.smb\Plex Media Server\", then the ls command only returns "Metadata" and "Plug-in Support"

And I tried this command with no success:
`u0_a103@localhost:/$ ls -l "/SDCard/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so"
ls: /SDCard/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so: No such file or directory`

**Comment by agneevX** [comment]
Not sure why you would have metadata stored in two separate places.
What does 
`ls -l "/SDCard/Android/data/com.plexapp.mediaserver.smb/Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv/Libraries/Android/aarch64/ucs2/apsw.so"` and 

`ls -l "/SDCard/Android/data/com.plexapp.mediaserver.smb/Plex Media Server"` output?

**Comment by TonioBDS** [comment]
oups sorry, its horrible !
I don't know how to do it better than this: 

![image](https://user-images.githubusercontent.com/49921859/78302568-55217e00-753b-11ea-87c0-37523880e4e6.png)

**Comment by TonioBDS** [comment]
> Not sure why you would have metadata stored in two separate places.
Normally all my metadata are on USBStick, not in internal drive of the shield. So your comment invted me to check what do I have on the metadata on the internal storage...

Actually it looks like this :
![image](https://user-images.githubusercontent.com/49921859/78303035-353e8a00-753c-11ea-95f7-ab5bd78fda50.png)
This means that when in SSH I go to SDCard folder, in fact I access to my internal storage and not on to the metadata on the USB stick...
Sorry for my poor level, I should have seen it before.

Then to access to my USB in SSH I stuck... ls command doesn't see it...
Thank you again.

**Comment by TonioBDS** [comment]
OK I progress: after connecting in SSH, it tried several "cd .." which revealed that I was like in the folder of the SSH application "SSHelper".
Then when I go in upper folder, I try ls -l and I see that I have no permission:
![image](https://user-images.githubusercontent.com/49921859/78303630-594e9b00-753d-11ea-8f01-5ff53ee5e5a2.png)

**Comment by agneevX** [comment]
I think this is an issue related to your non-conventional installation. Why do you store your metadata on a microSD card? It's much slower and wears it out much faster.

**Comment by TonioBDS** [comment]
Well I've read that the only 16Gb of internal drive could be very limitative with large plex Media due to metadata storage and temporary space during transcoding.
Moreover I have installed a high speed sd card on a usb3 adaptator.
Finally this was the ony way I fould to access in SMB to the plugin folder with the write permission and then copy the Trakt plugin files.
In any case this doesn't work.
